dracut/30coreos-installer/module-setup.sh

```diff
@@ -35,6 +35,19 @@ install() {
     inst_multiple /usr/sbin/blockdev
     inst_multiple /usr/sbin/wipefs
 
+    if [ "$arch" = "s390x" ]; then
+        inst_multiple -o /usr/bin/wc
```

Comment:
This is totally fine, though I'm curious, why use `inst_multiple` for single installs?

Reply:
I want to use `-o` option of `inst_multiple` which means, if the file does not exist, just ignore it. That was for when I didn't use the "$arch" check, still left it there. Will change and test again.

Reply:
Actually, `inst_multiple` was used multiple times in this same file. Even though the full path of the binaries were provided, I also don't know why `inst_multiple` is used from the beginning.  Since I don't want to break something I'm not aware of, I would prefer to keep it like this.